app/src/main/java/com/example/concordia_campus_guide/Models/WalkingPoint.java

```diff
@@ -48,11 +48,15 @@
     @TypeConverters(EnumToStringConverter.class)
     private PointType pointType;
 
-    public WalkingPoint(@NonNull Coordinates coordinate, @NonNull String floorCode, List<Integer> connectedPointsId, PointType pointType) {
+    @ColumnInfo(name = "place_code")
+    private String placeCode;
+
+    public WalkingPoint(@NonNull Coordinates coordinate, @NonNull String floorCode, List<Integer> connectedPointsId, PointType pointType, String placeCode) {
         this.coordinate = coordinate;
         this.floorCode = floorCode;
         this.connectedPointsId = connectedPointsId;
         this.pointType = pointType;
+        this.placeCode = placeCode;
```

Comment:
Why do we need both placeCode and floorCode? We should be able to access the floor from the place no?

Reply:
no, the place code is the id that maps the walkingpoint to a place. for rooms, it would return the roomCode (e.g 937) and for buildings it would return the buildingCode (e.g H-9). They are not combined as the name would suggest. I understand that this might be confusing, so do you have any other suggestions for the field's name?